tests/conftest.py

```diff
@@ -1,10 +1,15 @@
 # -*- coding: utf-8 -*-
+import os
+from pathlib import Path
+
 import pytest
 from responses import RequestsMock
 
 from mozci.push import MAX_DEPTH, Push
 from mozci.util.hgmo import HGMO
 
+os.environ["ADR_CONFIG"] = str(Path("tests/config.toml"))
+
```

Comment:
I don't think this will ultimately make a difference, but I'd probably use `monkeypatch` in an `autouse` fixture. Something like:
```
@pytest.fixture(autouse=True, scope="session")
def set_adr_config(monkeypatch):
    monkeypatch.setenv("ADR_CONFIG", str(Path("tests/config.toml"))
```

Reply:
You had looked at my draft PR. That did not work.